master/master.cfg

```diff
@@ -132,6 +132,7 @@ _WORKERS = [
 # worker name and password must be configured on the worker.
 c['workers'] = [Worker(n,
                        WORKER_SECRET,
+                       keepalive_interval=600,  # default is 3600 (1 hour). We'll do 10 mins.
```

Comment:
Should these not match?

Reply:
The defaults don't. (master -> worker 3600, worker->master 600)